tests.ts

```diff
@@ -0,0 +1,56 @@
+
+const functsFromMain = require('./main');
+const { readFile } = require('fs')
+const assert = require('assert');
+const { writeNumSymbolsInFileIntoFile, numOfSymbols, numVowels } = functsFromMain
+
+
+
+// test for writeNumSymbolsInFileIntoFile function
+describe("writeNumSymbolsInFileIntoFile", () => {
+    it("Should work without errors", (done) => {
+        // execute function
+        writeNumSymbolsInFileIntoFile('data.txt', 'result.txt', "l", (err: string) => {
+            if (err) {
+                done(err)
+            } else {
+                done()
+            }
+        })
+    })
```

Comment:
@Anton583 
Your thinking is right to make this test which tests that everything works without error. However, this test is not enough. You are not testing that the function does anything correctly, you are only checking that it doesn't crash, essentially.